### DISCUSSION TITLE
keplergl pypi uncontinued?

### DISCUSSION THREAD

**Discussion by Minou21** [keplergl/kepler.gl#2525] [Q&A]
Hello everyone :),

may I kindly ask if there are any plans to update the keplergl pypi variant? It is already quite outdated.
Is there an alternative existing? I could not find any information regarding this. Sorry if this is an obvious question.

Kindly, 
Viviane

**Comment by lloydbrodsky** [comment]
Just to be clear the current version on PyPi is 0.3.2 dated Sep 16 2021 so people using the  Python widget wouldn't be using the current version, which I gather was a serious upgrade
